include/ListView/ListViewDisplay.php

```diff
@@ -363,6 +363,8 @@ protected function buildActionsLink($id = 'actions_link', $location = 'top')
                 } else {
                     $menuItems[] = $this->buildBulkActionButton($location);
                 }
+            } else {
+                $menuItems[] = $this->buildBulkActionButton($location);
             }
```

Comment:
@jack7anderson7 I think we can remove the full ` if and else` and just leave `$menuItems[] = $this->buildBulkActionButton($location);` as done on https://github.com/salesagility/SuiteCRM/pull/8600. `show_action_dropdown_as_delete` Does not seem to be set anywhere on the full codebase apart from the property initialization on this class.

Reply:
@jack7anderson7 plus, this also seems to require a css fix. See following screenshot of the dropdown after applying the fix.
![legacy-dropdown-acls-css-fix](https://user-images.githubusercontent.com/1140849/149768925-e01b3242-82e8-41bb-93cd-5caed5dbac3d.png)

![legacy-dropdown-acls-css-fix-dev-tools](https://user-images.githubusercontent.com/1140849/149768974-6b67bf71-f253-4a23-bc1d-27f64edefcbb.png)

![legacy-dropdown-acls-css-fix-dev-tools-fixed](https://user-images.githubusercontent.com/1140849/149768987-36ed761d-9d45-4f53-a5f6-5e7ba2260915.png)

Reply:
Added an empty link item when there's no delete button, The other option is to split Print as PDF, Export and Map into seperate list items which requires a lot of changes to custom ListViewSmarty classes, or create custom css for the list.

Is deleting the show_action_dropdown_as_delete property ok to do?